Source/Fuse.Elements/Resources/Exif/Exif.uno

```diff
@@ -0,0 +1,196 @@
+using Uno;
+using Uno.IO;
+using Uno.Compiler.ExportTargetInterop;
+
+namespace Fuse.Resources.Exif
+{
+	/** ImageOrientation is used to indicate the orientation of an image.
+		The enum describes a transformation containing a vertical flip (if the FlipVertical flag is active), and then a rotation (if one of the rotation flags is active).
+		Note that the flip should be applied before the rotation.
+	 */
+	[Flags]
+	public enum ImageOrientation
+	{
+		Identity = 0,
+		Rotate90 = 1,
+		Rotate180 = 2,
+		Rotate270 = 3,
+		FlipVertical = 1 << 2,
+	}
+
+	public struct ExifData
+	{
+		public readonly ImageOrientation Orientation;
+
+		public static bool FromByteArray(byte[] buffer, out ExifData data)
+		{
+			if defined(Android)
+				data = ExifAndroidImpl.FromByteArray(buffer);
+			if defined(iOS)
+				data = ExifIOSImpl.FromByteArray(buffer);
+			if defined(DotNet)
+				data = ExifDotNetImpl.FromByteArray(buffer);
+			else
+			{
+				data = new ExifData(0);
```

Comment:
`data = default(ExifData);` instead?

Reply:
i'll do this and the one below as a separate PR